rust/scx_rustland_core/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "scx_rustland_core"
+version = "0.1.0"
+edition = "2021"
+authors = ["Andrea Righi <andrea.righi@canonical.com>"]
+license = "GPL-2.0-only"
+repository = "https://github.com/sched-ext/scx"
+description = "Framework to implement sched_ext schedulers running in user-space"
```

Comment:
FYI / completely optional nit: if we want to be grammatically precise, user-space should be written as "user space" here, and in other places where it's not being used as a compound adjective. A hyphen is only used like this when part of a compound adjective. In other words, to differentiate the meaning of the adjective from the words used separately, such as fire proof suit -> fire-proof suit, left handed boxer -> left-handed boxer, user space scheduler -> user-space scheduler, etc. Just user space on its own is a noun, not a compound adjective describing something like a type of scheduler.

Reply:
That is really interesting! Thanks for sharing it (will fix it).